applovin_max/lib/src/ad_classes.dart

```diff
@@ -39,6 +45,9 @@ class MaxAd {
   /// The placement name that you assign when you integrate each ad format.
   final String placement;
 
+  /// The latency of the mediation ad load request in milliseconds.
+  final double latency;
```

Comment:
We should always append the time unit to the API to ensure users know it is milliseconds. In our Android SDK, it's `long getLatencyMillis();`.

Reply:
I guess iOS is different though. But check out Unity Plugin's:

```
 public class AdInfo
    {
        ...
        public long LatencyMillis { get; private set; }
```